scripts/puppeteer/src/utils.ts

```diff
@@ -30,6 +30,32 @@ export async function launch(options: LaunchOptions = {}) {
   return browser;
 }
 
+/**
+ *
+ * Try out https://github.com/puppeteer/puppeteer/issues/1490
```

Comment:
ok this won't fix anything , pipeline failed 
<img width="1170" alt="image" src="https://user-images.githubusercontent.com/1223799/214117215-f133ebb3-0e6b-440f-904f-54be6192f2bf.png">